application/contributors-list.jsx

```diff
@@ -0,0 +1,98 @@
+        let owner = this.props.params.owner
+        ,   shortName = this.props.params.shortName;
+        fetch(pp + `api/repos/${owner}/${shortName}/contributors`, { credentials: "include" })
+            .then(utils.jsonHandler)
+            .then((data) => {
+                this.setState({ 
+                    substantiveContributors: data.substantiveContributors
+                    , nonSubstantiveContributors: data.nonSubstantiveContributors
+                    , owner: owner
+                    , shortName: shortName
+                    , status: "ready"
+                });
+            })
+            .catch(utils.catchHandler)
+        ;
+    }
+    
+    render () {
+        let st = this.state
+        ,   content
+        ,   repositoryLink = ""
+        ,   substantiveKeys
+        ,   nonSubstantiveKeys
+        ;
+        if (st.status === "loading") {
+            content = <Spinner prefix={pp}/>;
+        }
+        else if (st.status === "ready") {
+            repositoryLink = <a href={`https://github.com/${st.owner}/${st.shortName}`}>{`${st.owner}/${st.shortName}`}</a>;
+            substantiveKeys = Object.keys(st.substantiveContributors);
+            nonSubstantiveKeys = Object.keys(st.nonSubstantiveContributors);
+            content = (
+                <div>
+                    {[substantiveKeys, nonSubstantiveKeys].map((type) => {
+                        if (type.length > 0) {
+                            return (
+                                <div key={type}>
+                                    <h3>{type === substantiveKeys ? "Substantive" : "Non-substantive"} contributions</h3>
+                                    <table>
+                                        <thead>
+                                            <tr>
+                                                <th>Contributor</th>
+                                                <th>PR</th>
+                                            </tr>
+                                        </thead>
+                                        <tbody>
+                                            {
+                                                type.map((i) => {
+                                                    return <tr key={(type === substantiveKeys ? st.substantiveContributors : st.nonSubstantiveContributors)[i].name}>
+                                                        <td>{(type === substantiveKeys ? st.substantiveContributors : st.nonSubstantiveContributors)[i].name}</td>
+                                                        <td>
+                                                            <ul>
+                                                                {(type === substantiveKeys ? st.substantiveContributors : st.nonSubstantiveContributors)[i].prs.map((pr) => {
+                                                                    return <li key={pr}><a href={`${pp}pr/id/${st.owner}/${st.shortName}/${pr}`}>PR #{pr}</a></li>
```

Comment:
I suspect it would be more useful to link to the PR itself rather than the related IPR checker page for the PR; unless you had a specific reason to link to the latter?

Reply:
I wasn't sure which link would be the most useful but I guess people will be more interested to see the details of the PR instead of the IPR check.
That said, I'm wondering if we should also list the status of the PR (open, close, etc) on the list.

Reply:
excellent remark! it's clear what we care primarily about is *merged* pull requests; unfortunately, AFAIR, we only track a open/closed state, and not very reliably either (e.g. if a pull request gets open or closed when ash-nazg is down for any reason, there isn't any systematic sync).

Another thing that would be worth highlighting one way or another is pull requests that got merged despite not having received IPR-validation.

Let's merge and deploy this as is, and follow up in separate issues / PRs.

Reply:
I've created https://github.com/w3c/ash-nazg/issues/270 and https://github.com/w3c/ash-nazg/issues/271 and will tackle these soon.